src/lib/Libcmds/parse_at.c

```diff
@@ -182,7 +182,7 @@ parse_at_list(char *list, int use_count, int abs_path)
 			;
 
 		/* Drop any trailing blanks */
-		for (b = c - 1; isspace(*b); b--)
+		for (b = c - 1; b >= l && isspace(*b); b--)
```

Comment:
I would suggest the following:
```
for (b = c - 1; (b > 0) && isspace(*b); b--)
```

Reply:
This sanity check ensures that the address of `b` is never below the address of `l` which is the original string we are traversing.

Reply:
Sorry, that looked like a one. Please do add parenthesis around (b >= l)

Reply:
I'd consider using a different variable than a lowercase ell.  It just looks too much like a one.